src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -1501,4 +1502,64 @@ def do_request(params)
       end
     end
   end
+
+  describe 'POST #save_new' do
+    before do
+      login(user)
+    end
+
+    context 'valid package name' do
+      before do
+        post :save_new, params: { project: source_project, name: 'new-package',
+                                  title: 'package foo',
+                                  description: 'awesome package foo' }
+      end
+
+      it { expect(response).to have_http_status(:redirect) }
```

Comment:
Use better the `redirect_to` method and add the path to where is being redirected.